manager/controllers/app/select_modules.go

```diff
@@ -364,22 +372,65 @@ func validateClusterRestrictions(item *DataInfo, edge *ResolvedEdge, cluster mul
 }
 
 func validateClusterRestrictionsPerCapability(item *DataInfo, capability taxonomy.Capability, cluster multicluster.Cluster) bool {
-	restrictions := item.Configuration.ConfigDecisions[capability].DeploymentRestrictions.Clusters
+	return validateRestrictions(item.Configuration.ConfigDecisions[capability].DeploymentRestrictions.Clusters, &cluster)
+}
+
+func validateStorageRestrictionsPerCapability(item *DataInfo, capability taxonomy.Capability, account taxonomy.StorageAccount, bandwidth *adminrules.BandwidthMatrix) bool {
+	if !validateRestrictions(item.Configuration.ConfigDecisions[capability].DeploymentRestrictions.StorageAccounts, account) {
+		return false
```

Comment:
Please add printing in log for this

Reply:
No need, there is a printout at the caller level